devTools/docker/sync-s3-images.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env  bash
+
+# This script assumes that you have ssh access to the OWID production db server at live-db.owid.io It will
+# fail if you don't have these credentials
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
```

Comment:
It doesn't error out here as expected due to `-o nounset` because it gets set to empty in the default `.env` file. It would be nice to add a usability message like the one below.

```
if [[ -n "$IMAGE_HOSTING_BUCKET_PATH" ]]; then
  echo 'Please set IMAGE_HOSTING_BUCKET_PATH in .env'
  exit 1
fi
```